client/src/document/index.tsx

```diff
@@ -156,6 +157,8 @@ export function Document(props /* TODO: define a TS interface for this */) {
         </div>
       )}
 
+      {doc.isTranslated && <LocalizedContentNote doc={doc} locale={locale} />}
```

Comment:
```suggestion
      {doc.isTranslated && <LocalizedContentNote doc={doc} locale={locale} isActive={doc.isActive} />}
```
That way, the source of truth about whether a locale is active or not is canonical and not dependent on business logic within that molecule. 
And within the molecule, use en-US as a fallback text if we don't have a translation.